.vscode/settings.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
This should be removed along with .pyc files and pyronearEngine/raspberryPi/model/model_resnet18.txt and `test/.coverage`. Once removed, you might want to add a gitignore file similar to this one to avoid this later on (https://github.com/pyronear/PyroNear/blob/master/.gitignore) ;)